### PR TITLE
Tests/command item

### PR DESCRIPTION
**Pull request by mahboubii**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: src/components/CommandItem/__tests__/CommandItem.test.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import '@testing-library/jest-dom';
+
+import CommandItem from '../CommandItem';
+
+afterEach(cleanup); // eslint-disable-line
+
+describe('commandItem', () => {
+  it('should render component with empty entity', () => {
+    const tree = renderer.create(<CommandItem entity={{}} />).toJSON();
+    expect(tree).toMatchInlineSnapshot(`
```
Comment (vishalnarkhede) [inline src/components/CommandItem/__tests__/CommandItem.test.js:13]: Can we keep snapshots in separate files? similar to this - https://github.com/GetStream/stream-chat-react/pull/236/files#diff-9f8593107b8d0b64260cd6c820bef430R39

Comment (mahboubii) [reply src/components/CommandItem/__tests__/CommandItem.test.js:13]: they suggest using inline snapshots whenever the snapshot is short but fine with me if you think it's better to put them in a separate file 
https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/prefer-inline-snapshots.md

File: src/components/CommandItem/__tests__/CommandItem.test.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      </div>
+    `);
+  });
+
+  it('should render component with custom entity prop', async () => {
+    const entity = { name: 'name', args: 'args', description: 'description' };
+    const Component = <CommandItem entity={entity} />;
+
+    const { getByText } = render(Component);
+    await waitFor(() => {
```
Comment (ath92) [inline src/components/CommandItem/__tests__/CommandItem.test.js:36]: Do you need the waitFor here? I was under the impression that `render` is synchronous

Comment (mahboubii) [reply src/components/CommandItem/__tests__/CommandItem.test.js:36]: you're right it's not needed

**Pull request merged**